src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveSummaryService.kt

```diff
@@ -200,12 +221,12 @@ class NoPrisonersAtLocationException(prisonId: String, locationId: String) :
 data class IepResult(
   val bookingId: Long,
   val iepLevel: String,
-  val daysSinceReview: Int,
-  val daysOnLevel: Int
+  val daysSinceReview: Int?,
```

Comment:
correct me if i'm wrong, but these are nullable only because our DB might not have any "genuine reviews" but does have an "initial" record?

Reply:
correct!